### DISCUSSION TITLE
cosmolgy is not dead yet

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#168] [Cosmology Models]
“Big Bang not yet dead but in decline” Nature 1995
https://www.nature.com/articles/377099a0

"ΛCDM not dead yet: massive high-z Balmer break galaxies are less common than previously reported"
https://arxiv.org/abs/2310.03063

Cosmology is not healthy - it's been on artificial life support for decades.

**Comment by OlliSant** [comment]
One old galaxy there is enough to ruin BB. No enough time to evolve to such a galaxy from the supposed "beginning" of the universe. They say there is not so many galaxies.

**Comment by bgreenman** [comment]
BB is dying, but there hasn't been major efforts in mainstream academia to truly look for alternatives. some alternative researchers in academia have started looking for alternatives, but have not found widespread acceptence in their fields of research.
